src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -60,6 +63,20 @@ namespace {
 constexpr uint8_t kDACCertificate = 1;
 constexpr uint8_t kPAICertificate = 2;
 
+#ifdef ENABLE_HSM_EC_KEY
+class OpCred_P256Keypair : public Crypto::P256KeypairHSM
+{
+public:
+    OpCred_P256Keypair()
+    {
+        SetKeyId(CASE_OPS_KEY);
+        provisioned_key = true;
+    }
+};
+#else
+using OpCred_P256Keypair = Crypto::P256Keypair;
+#endif
```

Comment:
This seems to imply there can only be 1 operational key pair. How do you tell apart the operational keypairs from different fabrics?

Also, this code very strongly makes assumptions about one crypto backend in common code. We should avoid this. Isn't there a better method to steer the key usage?

Reply:
1. The integration is intended for end node side. Every time the node tries to join a fabric, new key is generated in HSM. (only the key id remains same).

2. Do you prefer to move the above declarations of OpCred_P256Keypair to crypto layer ?